### ISSUE TITLE
cp fails, folder treated as a flag

### ISSUE THREAD

**Issue by roblabla** [sagiegurari/shell2batch#2]
**Describe the bug**
cp seems to be coverted to xcopy, which doesn't support forward slashes to separate path items. As a result, running `cp linker-scripts/bootstrap.ld link.T` results in:

```
xcopy linker-scripts/bootstrap.ld link.T
Invalid switch - /bootstrap.ld
```

**To Reproduce**

**Code Sample**
```rust
/// paste code here
```

**Comment by roblabla** [comment]
This seems to be a pitfall in almost all the commands actually. move also accepts only `\`-delimited path items for instance. I'm thinking of adding an extra flag to convert_line's big match, "convert_paths", that, if true, will basically replace all `/` with `\`.

**Comment by sagiegurari** [comment]
thanks, will fix

**Comment by sagiegurari** [comment]
just released new 0.3.0 version with this fix for specific commands.
i'll be releasing a new cargo-make version which will use this new shell2batch as well soon

**Comment by sagiegurari** [comment]
going to close this for now. if there are more issues just reopen or open a new issue.
